boards/shields/weact_ov2640_cam_module/doc/index.rst

```diff
@@ -0,0 +1,82 @@
+.. _weact_ov2640_cam_module:
+
+WeAct Studio MiniSTM32H7xx OV2640 Camera Sensor
+###############################################
+
+Overview
+********
+
+The OV2640 camera sensor is designed to interface with the WeAct Studio
+MiniSTM32H7xx boards, providing camera sensor capabilities. This shield
+integrates the OV2640 camera module, which is capable of capturing images
+and video with a resolution of up to 2 megapixels.
+
+.. figure:: ov2640.webp
+   :align: center
+   :alt: OV2640 Camera Sensor
+
+More information about the OV2640 camera sensor can be found on the
+`MiniSTM32H7xx GitHub`_ and in the `OV2640 datasheet`_.
+
+Requirements
+************
+
+Your board needs to have a ``dcmi`` device tree label to work with this shield.
```

Comment:
Is it now `zephyr_camera_dvp`?
https://github.com/zephyrproject-rtos/zephyr/compare/d73a2d869c5c426f64aea6860b63205a42a1fbbf..76759c81c076ea47aebe244b2f5daa940ae32eca#diff-a04707eebe85f0f2eef1831b4061cffe823f57bbf4a2ac8f549d08a167f3e106L28

Maybe it was to mean that it is only expected to work with DCMI in the current state... in which case "devicetree label".

Also, `device tree` -> `devicetree` according to this search, also also used interchangeably it seems...
https://docs.zephyrproject.org/latest/gsearch.html?q=device+tree&check_keywords=yes&area=default#gsc.tab=0&gsc.q=device%20tree&gsc.page=1